docs/.vitepress/config.ts

```diff
@@ -59,170 +49,116 @@ export default defineConfig({
-                { text: 'wxt', link: 'wxt.md' },
-                { text: 'wxt build', link: 'wxt-build.md' },
-                { text: 'wxt zip', link: 'wxt-zip.md' },
-                { text: 'wxt prepare', link: 'wxt-prepare.md' },
-                { text: 'wxt clean', link: 'wxt-clean.md' },
-                { text: 'wxt init', link: 'wxt-init.md' },
-                { text: 'wxt submit', link: 'wxt-submit.md' },
-                { text: 'wxt submit init', link: 'wxt-submit-init.md' },
-              ],
-            },
-            {
-              text: 'API Reference',
-              collapsed: true,
-              items: filteredTypedocSidebar,
-            },
-          ],
-        },
-      ],
-    },
+      '/get-started/': menuRoot([
+        menuGroup('Get Started', '/get-started/', [
+          menuItem('Introduction', 'introduction'),
+          menuItem('Installation', 'installation'),
+          menuItem('Configuration', 'configuration'),
+          menuItem('Entrypoints', 'entrypoints'),
+          menuItem('Assets', 'assets'),
+          menuItem('Testing', 'testing'),
+          menuItem('Publishing', 'publishing'),
+          menuItem('Migrate to WXT', 'migrate-to-wxt'),
+          menuItem('Compare', 'compare'),
+        ]),
+      ]),
+      '/guide/': menuRoot([
+        menuGroup('Key Concepts', '/guide/key-concepts/', [
+          menuItem('Manifest', 'manifest'),
+          menuItem('Auto-imports', 'auto-imports'),
+          menuItem('Web Extension Polyfill', 'web-extension-polyfill'),
+          menuItem('Frontend Frameworks', 'frontend-frameworks'),
+          menuItem('Content Script UI', 'content-script-ui'),
+        ]),
+        menuGroup('Directory Structure', '/guide/directory-structure/', [
+          // Folders
+          menuItem('.output/', 'output'),
+          menuItem('.wxt/', 'wxt'),
+          menuItem('assets/', 'assets'),
+          menuItem('components/', 'components'),
+          menuItem('composables/', 'composables'),
+          menuGroup('entrypoints/', '/guide/directory-structure/entrypoints/', [
+            menuItem('background', 'background.md'),
+            menuItem('bookmarks', 'bookmarks.md'),
+            menuItem('*.content.ts', 'content-scripts.md'),
+            menuItem('*.css', 'css.md'),
+            menuItem('devtools', 'devtools.md'),
+            menuItem('history', 'history.md'),
+            menuItem('newtab', 'newtab.md'),
+            menuItem('options', 'options.md'),
+            menuItem('popup', 'popup.md'),
+            menuItem('sandbox', 'sandbox.md'),
+            menuItem('sidepanel', 'sidepanel.md'),
+            menuItem('*.html', 'unlisted-pages.md'),
+            menuItem('*.ts', 'unlisted-scripts.md'),
+          ]),
+          menuItem('hooks/', 'hooks'),
+          menuItem('modules/', 'modules'),
+          menuGroup('public/', 'public', [
+            menuItem('_locales/', 'public/locales'),
+          ]),
+          menuItem('utils/', 'modules'),
```

Comment:
Mistake from my previous PR.

```suggestion
          menuItem('utils/', 'utils'),
```